spec/requests/api/v1/registrations_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'Api::V1::Registrations', type: :request do
+
+  let(:email)                 { 'test@test.com' }
+  let(:password)              { '123456' }
```

Comment:
i recommend you install factory bot gem, it will help you to create data for rspec